tests/roles/dataplane_adoption/defaults/main.yaml

```diff
@@ -4,3 +4,4 @@ image_tag: "current-podified"
 ansibleSSHPrivateKeySecret: dataplane-adoption-secret
 deployStrategy:
   deploy: false
+edpm_encoded_privatekey: "{{ edpm_privatekey|b64encode }}"
```

Comment:
so edpm_privatekey will no longer be a thing, after we merge the rdo-jobs side https://review.rdoproject.org/r/c/rdo-jobs/+/49298

will you followup? but what will the default be?
 this is a _required_ var right so perhaps we should not even set a default?

Reply:
I needed this because I have to encode `edpm_privatekey` , I plan to remove this line after  https://review.rdoproject.org/r/c/rdo-jobs/+/49298 gets merged

Reply:
ok thanks sounds good